src/stories/components/buttons.js

```diff
@@ -1,17 +1,107 @@
 import React from 'react';
+import styled from 'styled-components';
 import Button from 'components/buttons/Button';
-import ButtonWebUSB from 'components/buttons/WebUsb';
 import ButtonPin from 'components/buttons/Pin';
 import ButtonNotification from 'components/buttons/Notification';
+import { H1 } from 'components/Heading';
 import { storiesOf } from '@storybook/react';
 import {
     withKnobs, text, boolean, select,
 } from '@storybook/addon-knobs';
 import { withInfo } from '@storybook/addon-info';
+import { linkTo } from '@storybook/addon-links';
 
 import colors from 'config/colors';
+import icons from 'config/icons';
 
-storiesOf('Components', module)
+const Wrapper = styled.div`
+    padding: 1.6rem;
+`;
+
+const Row = styled.div`
+    display: flex;
+    justify-content: space-evenly;
+    padding: 1rem 0;
+    margin: .5rem 0 2rem;
+`;
+
+const BtnLink = styled.button`
+    font-size: 1rem;
+    color: ${colors.TEXT_SECONDARY};
+    vertical-align: middle;
+    background: ${colors.LANDING};
+    padding: .5rem;
+    border: none;
+    border-radius: 5px;
+`;
+
+storiesOf('Buttons', module)
+    .add('All', () => (
+        <Wrapper>
+            <H1>
+                Default <BtnLink onClick={linkTo('Buttons', 'Button')}>{'<Button />'}</BtnLink>
```

Comment:
I couldn't figured out that this is clickable link, can we add `cursor:pointer` on` BtnLink`?

Reply:
added cursor:pointer and hover effect